### PR TITLE
Reactconf challenge

### LINKED ISSUE #20: ReactConfBR ticket
Who sends the pull request first to close this itens, will win a ticket for ReactConfBR(http://reactconfbr.com.br/)

- [ ] create an UserRow
- [ ] use a createFragmentContainer to define UserRow requirements
- [ ] create createQueryRenderer utils (based on this https://github.com/entria/relay-utils)

### PR DESCRIPTION
**Pull request by brunolemos**
Closes #20 

- [x]  create an UserRow
- [x]  use a createFragmentContainer to define UserRow requirements
- [x]  create createQueryRenderer utils (based on this https://github.com/entria/relay-utils)

### CODE REVIEW

File: src/UserRow.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    fragment UserRow_query on Query {
+      user(id: $id) {
+        name
+      }
+    }
+  `
+)
+
+// UserRowQueryRenderer
+const UserRowQueryRenderer = ({ userId, ...props }) => {
```
Comment (sibelius) [inline src/UserRow.js]: use QueryRenderer only on root component

https://facebook.github.io/relay/docs/query-renderer.html

Comment (brunolemos) [reply src/UserRow.js]: @sibelius where do you mean?

Comment (sibelius) [reply src/UserRow.js]: on UserList

UserRow fragment should be on UserList fragment

Comment (brunolemos) [reply src/UserRow.js]: @sibelius updated

File: src/UserRow.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <View style={styles.container}>
+          <Text>{user.name}</Text>
+        </View>
+      </TouchableHighlight>
+    )
+  }
+}
+
+// UserRowFragmentContainer
+createFragmentContainer(
```
Comment (sibelius) [inline src/UserRow.js]: export this one

File: src/UserRow.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  UserRow,
+  graphql`
+    fragment UserRow_query on Query {
```
Comment (sibelius) [inline src/UserRow.js]: you wanna a fragment on User instead

**Comment by sibelius** [comment]
great job

**Pull request closed without merging**